index.ftd

```diff
@@ -456,28 +459,46 @@ ftd.image-src src:
 optional ftd.align alignment: center
 optional ftd.resizing width:
 optional ftd.resizing height:
+optional ftd.image-src src2:
+optional boolean blog: false
 
 -- ftd.column:
 width: fill-container
 align-content: center
-height: $image.height
+
+-- ftd.row:
+spacing.fixed.px: 26
 
 -- ftd.image:
 src: $image.src
 width: auto
 width if { image.width != NULL }: $image.width
 max-width: fill-container
-height: auto
+height if { image.height != NULL }: $image.height
 align-self: center
 
+-- ftd.image:
+if: { image.src2 != NULL }
+src: $image.src2
```

Comment:
@saurabh-lohiya give meaningful property name, what do you mean by src2? is this the way world will use it?